<launch>

	<!-- <node pkg="rostopic"
		type="rostopic"
		name="start_sim"
		args="pub /startSimulation std_msgs/Bool true" /> -->


	<node name="coppelia_combiner" pkg="stewart_platform" type="coppelia_combiner" args=""
		respawn="false"
		output="screen">

		<!-- if false, it work on coppeliasim with rosInterface plugin-->
		<param name="old_plugin" type="bool" value="false" />

		<!-- subscribers topic-->
		<param name="camera_color_name" type="string" value="/camera/color/image_rect_color" />
		<param name="camera_depth_name" type="string"
			value="/camera/aligned_depth_to_color/image_raw" />
		<param name="depth_buffer_in" type="string" value="/coppelia/depth_buff" />
		<param name="point_cloud_out" type="string" value="/camera/depth_registered/points" />


		<param name="u_res" value="480" />
		<param name="v_res" value="640" />
		<param name="near_clip" value="1.00e-02" />
		<param name="far_clip" value="5.00e+00" />
		<param name="view_angle" value="60.0" />
	</node>


	<node name="colorSeg" pkg="stewart_platform" type="color_segmentation" args="" respawn="false"
		output="screen">

		<param name="cld_topic_name" type="string" value="/camera/depth_registered/points" />
		<param name="fixed_frame" type="string" value="world" />
		<param name="optical_frame" type="string" value="camera_color_optical_frame" />

	</node>

	<!-- <node pkg="tf" type="static_transform_publisher" name="base_2_world" output="screen" args="0 0 0 0 0 0 /world /robot_base 1"/>
	<param name="robot_description" command="cat $(find stewart_platform)/urdf/stewart_platform.urdf" /> -->

	<node name="stewart_platform" pkg="rviz" type="rviz"
		args="-d $(find stewart_platform)/rviz/stewart.rviz" respawn="false" />


</launch>	
    